settings.gradle

```diff
@@ -192,3 +194,27 @@ if (JavaVersion.current() == JavaVersion.VERSION_1_8) {
   }
 }
 
+dependencyResolutionManagement {
+  versionCatalogs {
+    libs {
+      from(files("./libs.versions.toml"))
```

Comment:
If you move `libs.versions.toml` to the `gradle/` directory, you can omit this one.